src/components/TimerModal/TimerBundle.js

```diff
@@ -0,0 +1,32 @@
+import {useContext, useEffect} from "react";
+import {PlayContext, TimerContext} from "./TimerContextProvider";
+import Timer from "./Timer";
+import TimerPlay from "./TimerPlay";
+import style from "./Modal.module.scss";
```

Comment:
We shouldn't be importing styles like this. See how styles have been applied to the global components for reference. You just need to add an import statement to the index.scss